### PR TITLE
Added tests

### PR DESCRIPTION
**Pull request by poratuk**

### CODE REVIEW

**Comment by poratuk** [comment]
This task are not completed:
please look on https://github.com/poratuk/pho-server-rest/blob/complete-tests/tests/NodeRoutesTest.php#L75 

I do not shure how must be added Edge and what parameters must be sended.

File: tests/NodeRoutesTest.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    public function testGetOutEdge()
+    {
+        $res = $this->get('/' . $this->founder_id . '/edges/in', true);
+        $this->assertEquals(200, $res->getStatusCode());
+
+        $body = json_decode($res->getBody(), true);
+        $this->assertSame($body, []);
+    }
+
+    public function testGetNonExsitingEdge()
```
Comment (esokullu) [inline tests/NodeRoutesTest.php]: Existing, not Exsiting

File: tests/EntityRoutesTest.php [hunk 2]
```diff
@@ -9,9 +9,86 @@
  * file that was distributed with this source code.
  */
 
-class EntityRoutesTest extends TestCase 
+class EntityRoutesTest extends TestCase
 {
+    public $founder_id = '';
```
Comment (esokullu) [inline tests/EntityRoutesTest.php]: why public? let's keep privileges as minimal as possible.

File: tests/NodeRoutesTest.php [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    {
+        $this->expectException('\GuzzleHttp\Exception\ServerException');
+        $res = $this->get('/' . $this->founder_id . '/edges/nonExist', true);
+    }
+
+    public function testCreateEdge()
```
Comment (esokullu) [inline tests/NodeRoutesTest.php:73]: what about "like" that i told you?

File: tests/TestCase.php [hunk 4]
```diff
@@ -9,27 +9,51 @@
...
             return $res;
+        }
+
         $body = json_decode($res->getBody(), true);
         return $body;
     }
 
-}
\ No newline at end of file
+    protected function post(string $path, array $postData)
```
Comment (esokullu) [inline tests/TestCase.php]: i would add $headers similarly to the get function and make them behave similarly, for consistency purposes.

File: tests/TestCase.php [hunk 5]
```diff
@@ -9,27 +9,51 @@
...
+    {
+        $res = $this->client->request('POST', self::HOST . $path, ['form_params' => $postData]);
+        return $res;
+    }
+
+    protected function delete(string $path, array $postData = [])
+    {
```
Comment (esokullu) [inline tests/TestCase.php]: same as above...

**Review by esokullu — CHANGES_REQUESTED** [review]

**Review by esokullu — APPROVED** [review]

**Pull request merged**